libs/wai-utilities/src/Network/Wai/Utilities/Server.hs

```diff
@@ -271,6 +274,38 @@ emitLByteString lbs = do
     -- | Emit the bytestring on the first read, then always return "" on subsequent reads
     return . atomically $ swapTVar tvar mempty
 
+-- | Run the 'Applicatino'; check the response status; if >=500, throw a 'Wai.Error' with
```

Comment:
`Applicatino` 🤓

Reply:
heh.  let's make a bunch of PRs and fix that spelling error in wai/warp!  :-)